src/signals/incident-management/containers/LocationForm/components/Form/index.js

```diff
@@ -0,0 +1,137 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FormBuilder, FieldGroup, Validators } from 'react-reactive-form';
+import { isEqual } from 'lodash';
+
+import mapLocation from 'shared/services/map-location';
+import FieldControlWrapper from '../../../../components/FieldControlWrapper';
+import MapInput from '../../../../components/MapInput';
+import HiddenInput from '../../../../components/HiddenInput';
+
+import './style.scss';
+
+class Form extends React.Component { // eslint-disable-line react/prefer-stateless-function
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      location: props.location,
+      newLocation: props.newLocation,
+      locationForm: FormBuilder.group({
```

Comment:
Moet dit wel in de state opgeslagen worden?

Reply:
bedoel je het locationForm.

ja daar ben ik mee begonnen, want er werd steeds in this.form de form state bewaard. maar het is toch eigenlijk de bedoeling om alles in props of state te bewaren. dan kun je er ook makkelijker testen.

Reply:
🤔 
Props en state zijn beide variabel. `locationForm` is dat niet; het moet niet aanpasbaar zijn en het verandert niet tijdens de lifecycle van het `Form` component. Je kunt het laten staan, maar het lijkt mij niet dat `locationForm` in de state aanwezig moet zijn.

Reply:
ja als je het zo bekijkt niet ok. ik ga het uit de state halen

Reply:
gedaan